### PR TITLE
Include peerpod specific crd deployments via operator

### PR DESCRIPTION
**Pull request by bpradipt**
This includes peerpod and peerpodconfig controllers
This is based on @caavery work. I just rebased and adapted the code to align with latest code base

### CODE REVIEW

File: config/overlays/peerpods/crd/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- github.com/confidential-containers/cloud-api-adaptor//peerpodconfig-ctrl/config/crd?ref=v0.7.0
+- github.com/confidential-containers/cloud-api-adaptor//peerpod-ctrl/config/crd?ref=v0.7.0
```
Comment (mythi) [inline config/overlays/peerpods/crd/kustomization.yaml:6]: Use the latest?
```suggestion
- github.com/confidential-containers/cloud-api-adaptor//peerpodconfig-ctrl/config/crd?ref=v0.8.0
- github.com/confidential-containers/cloud-api-adaptor//peerpod-ctrl/config/crd?ref=v0.8.0
```

File: go.mod [hunk 2]
```diff
@@ -3,6 +3,8 @@ module github.com/confidential-containers/operator
 go 1.20
 
 require (
+	github.com/confidential-containers/cloud-api-adaptor/peerpod-ctrl v0.7.1-0.20230803085141-495d3dfed9d8
+	github.com/confidential-containers/cloud-api-adaptor/peerpodconfig-ctrl v0.7.1-0.20230803085141-495d3dfed9d8
```
Comment (mythi) [inline go.mod:7]: ditto
```suggestion
	github.com/confidential-containers/cloud-api-adaptor/peerpod-ctrl v0.8.0
	github.com/confidential-containers/cloud-api-adaptor/peerpodconfig-ctrl v0.8.0
```

**Review by mythi — COMMENTED** [review]
LGTM but one question about which version to use. We could use 0.8.0, right?

File: go.mod (hunk 2 above)
Comment (bpradipt) [reply go.mod:7]: Thanks, I'll update it..

**Comment by bpradipt** [comment]
> LGTM but one question about which version to use. We could use 0.8.0, right?

We can use 0.8.0.

**Comment by bpradipt** [comment]
@mythi looks like you'll need to explicitly approve the PR for the approval to register

**Review by mythi — APPROVED** [review]

**Comment by mythi** [comment]
> @mythi looks like you'll need to explicitly approve the PR for the approval to register

another approval is needed still. perhaps @fidencio can take a look and judge if the failing tests are acceptable

**Comment by stevenhorsman** [comment]
The tests passed in the last nightly https://github.com/confidential-containers/operator/actions/runs/7838705386, so I'll re-queue in case that helps

**Review by jensfr — APPROVED** [review]
still looks good to me. Thanks Cathy and Pradipta!

**Pull request merged**